### DISCUSSION TITLE
Is anyone used Bun.js ?

### DISCUSSION THREAD

**Discussion by 0bill** [TechEmpower/FrameworkBenchmarks#7543] [General]
Hi Guys,
I have found interesting article that was promoting new framework [Bun.JS](https://bun.sh/)
Well, it's in beta, but I was wondering how much better it works than Node.js compared to what they describe on their website.
![image](https://user-images.githubusercontent.com/90815515/184363388-b4b93986-7b12-44e0-8b4b-4d3c070bff4e.png)

**Comment by cirospaciari** [comment]
@0bill 
Bun uses the uWebSockets C API same as https://github.com/uNetworking/uWebSockets.js, https://github.com/cirospaciari/socketify.py and https://github.com/cirospaciari/socketify.rb

Someone will add a bun benchmark soon, Bun still is in very early days.
I hope socketify.py https://github.com/TechEmpower/FrameworkBenchmarks/pull/7650  gets approved to next round
And someone just add uwebsockets.js https://github.com/TechEmpower/FrameworkBenchmarks/pull/7559 but needs approval too
